### PR TITLE
GUI-2388: Various fixes for availability zone fetches

### PR DESCRIPTION
**Pull request by kamalgill**
Fixes https://eucalyptus.atlassian.net/browse/GUI-2388

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1361/
Test FAILed.

**Comment by dkavanagh** [comment]
I'll bet there's a single root cause. I would have written 1 bug and fixed that. I'll go look at the root cause.

**Comment by dkavanagh** [comment]
I confirmed. this is actually fixed in the other PR.

**Comment by kamalgill** [comment]
@dkavanagh: I wrote separate tickets since I observed those issues w/o knowing the root cause.  Looking at the first line of PR 1759, I don't see that same fix when looking up the default_region in layout.py.  Are you sure that's a comprehensive fix?

**Comment by dkavanagh** [comment]
I ran through 3 of the cases 3 tickets outlines and those were fixed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1364/
Test FAILed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1365/
Test FAILed.

**Comment by aliceh** [comment]
Retest this please

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1371/
Test FAILed.

**Comment by aliceh** [comment]
Retest this please

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1375/
Test FAILed.

**Comment by aliceh** [comment]
Retest this, please

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1377/
Test FAILed.

File: eucaconsole/forms/volumes.py [hunk 1]
```diff
@@ -209,11 +208,10 @@ def __init__(self, request, conn=None, **kwargs):
         super(VolumesFiltersForm, self).__init__(request, **kwargs)
         self.request = request
         self.choices_manager = ChoicesManager(conn=conn)
-        region = request.session.get('region')
-        self.zone.choices = self.get_availability_zone_choices(region)
+        self.zone.choices = self.get_availability_zone_choices(self.region)
```
Comment (dkavanagh) [inline eucaconsole/forms/volumes.py]: Minor point, but some scalinggroups.py above, you chose to not pass region into the get_availabiity_zone_choices() method and here you still do.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1380/
Test FAILed.

**Pull request merged**